core/src/martian/encoders.cljc

```diff
@@ -36,6 +37,9 @@
   ([] (default-encoders keyword))
   ([key-fn]
    (merge
+    #?(:bb
+       {"application/transit+json"    {:encode #(transit-encode % :json)
+                                       :decode #(transit-decode % :json)}})
     #?(:clj
        {"application/transit+msgpack" {:encode #(transit-encode % :msgpack)
```

Comment:
We still need transit+json support for `clj` (but it needs to come after transit+msgpack)